tests/testthat/test_pension_supplement.R

```diff
@@ -0,0 +1,49 @@
+context("Pension Supplement")
+
+test_that("Warnings and Errors", {
+  expect_warning(pension_supplement(),
+                 reg_exp = '`Date` and `fy.year` not set, so using `Date` = "2016/03/01" and `fy.year` = "2015-16"')
+  expect_warning(pension_supplement(fy.year = "2015-16"),
+                 reg_exp = '`Date` not set. Using date as defined in fy2date()')
+  expect_warning(pension_supplement(Date = as.Date("2016-07-01"), fy.year = "2016-17"),
+                 reg_exp = "`fy.year` and `Date` both used. Ignoring `fy.year`.")
+  expect_error(suppressWarnings(pension_supplement(per = 'month')),
+               reg_exp = "per can only take values `fortnight` or `annual`")
+  expect_error(suppressWarnings(pension_supplement(has_partner = FALSE, seperated_couple = TRUE)),
+               reg_exp = "incompatible values of `has_partner` and `partner_seperated`")
+})
+
+test_that("Rates", {
+  #Max rates
+  expect_equal(suppressWarnings(pension_supplement(has_partner = FALSE)),
+            65)
+  expect_equal(suppressWarnings(pension_supplement(has_partner = TRUE)),
```

Comment:
`suppressWarnings` should be used sparingly. Usually a function should be runnable without any warnings.